SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security policy
+
+Security is of paramount importance to the Parsec project. We do all we can to identify and fix
+issues, however some problems might slip through the cracks. Any efforts towards responsible
+disclosure of security problems are greatly appreciated and your contributions will be acknowledged.
+
+## Supported versions
+
+Currently only the most recent version of the Parsec service is eligible for patching. This could
+change once a first stable version (`1.0.0`) is reached.
```

Comment:
Should the first stable version be `0.8.0`?

Reply:
Ideally they are all stable, as in we should not have done any breaking changes. I guess from `0.8.0`, stabiltiy is going to be checked but that does not mean that it was not stable before.
Maybe just say "This could change in the future"?